test/views/MentionsAndSentiments/test.index.js

```diff
@@ -0,0 +1,131 @@
+                            'key':'positive',
+                            'matching_results':49
+                          },
+                          {
+                            'key':'negative',
+                            'matching_results':21
+                          },
+                          {
+                            'key':'neutral',
+                            'matching_results':4
+                          }
+                        ]
+                      }
+                    ]
+                  },
+                ]
+              }
+            ]
+          },
+          {
+            'key':'Company Two',
+            'matching_results':465,
+            'aggregations':[
+              {
+                'type':'timeslice',
+                'field':'blekko.chrondate',
+                'interval':'1d',
+                'results':[
+                  {
+                    'key_as_string':'1488326400',
+                    'key':1488326400000,
+                    'matching_results':4,
+                    'aggregations':[
+                      {
+                        'type':'term',
+                        'field':'docSentiment.type',
+                        'results':[
+                          {
+                            'key':'positive',
+                            'matching_results':4
+                          }
+                        ]
+                      }
+                    ]
+                  },
+                  {
+                    'key_as_string':'1488412800',
+                    'key':1488412800000,
+                    'matching_results':43,
+                    'aggregations':[
+                      {
+                        'type':'term',
+                        'field':'docSentiment.type',
+                        'results':[
+                          {
+                            'key':'positive',
+                            'matching_results':1000000
+                          },
+                          {
+                            'key':'negative',
+                            'matching_results':3
+                          },
+                          {
+                            'key':'neutral',
+                            'matching_results':1
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                ]
+              }
+            ]
+          },
+        ]
+      }
+    ]
+  };
+
+  let queryAAA = {
+    date: {
+      from: '20170301',
+      to: '20170501'
+    },
+    text: 'Sample Company'
+  };
+
+  it('Orders results by number of "mentions"', () => {
+
+    let wrapper = shallow(<MentionsAndSentiments query={queryAAA} mentions={mentionsAAA} />);
+    let firstMention = wrapper.find(Accordion).nodes[0].props.header.props.children[0].props.children;
+    let secondMention = wrapper.find(Accordion).nodes[1].props.header.props.children[0].props.children;
```

Comment:
These definitely can't be looked up via class name? Like, if you do `wrapper.find(Accordion).debug()` do you see class name attrs anywhere in the logged tree?